Hyper/PlayerData/InventorySystem/Items/Pickaxes/Pickaxe.cs

```diff
@@ -22,12 +23,29 @@ public override void Use(Scene scene, IChunkWorker chunkWorker, float time)
         if (!chunkWorker.IsUpdating)
         {
             var location = scene.Player.GetRayEndpoint(in scene.SimulationManager.RayCastingResults[scene.Player.RayId]);
+            Vector3? otherSphereLocation = null;
```

Comment:
There is a ton of overlap in Use and Secondary use. Maybe try to remove it. I did something simmilar in the other PR for Chunk Mine/Build

Reply:
I did some delegates extravaganza to get rid of if statements. This shouldn't impact the performance too much because we're using reflection only once, but if you feel like it's too much, I can change it into ifs

Reply:
I think ifs would be better. I think I will change it anyways in my PR. I dont think chunks need Mine and Build. Just one method with some bool and here we could use that new method. (I already changed the chunk methods so no need to do that here)